### ISSUE TITLE
Version 2.1.2 is not installable

### ISSUE THREAD

**Issue by INblau-LucaGerhardt** [ratepay/magento2-module#79]
When running `bin/magento setup:upgrade` on version 2.1.2, following error occurs:

```
The XML in file "/var/www/html/vendor/ratepay/magento2-payment/etc/db_schema.xml" is invalid:
Element 'column', attribute 'identity': The attribute 'identity' is not allowed.
Line: 55

Verify the XML and try again.
```

Installed Magento Community Version: "2.4.3-p1"

**Comment by MarvinmStage** [comment]
I had the same problem with version 2.1.2 in Magento 2.4.2.
The problem is that the profile_id in line 55 is of type varchar, for which the auto_increment ("identity") cannot be used.
`<column xsi:type="varchar" name="profile_id" nullable="false" length="32" identity="true"/>`
After removing the attribute "identitfy='true'" from line 55, setup:upgrade is working fine.

**Comment by MartinKuhl** [comment]
we will analyse the issue. At the moment we are not able to reproduce the error. Can you give more details on your environment and on your update path (from which module version are you coming?) @MarvinmStage @INblau-LucaGerhardt

**Comment by MarvinmStage** [comment]
We updated from Version 1.2.9 to 2.1.2.

**Comment by MartinKuhl** [comment]
> We updated from Version 1.2.9 to 2.1.2.

any further information of your environment ?

**Comment by MarvinmStage** [comment]
Magento 2.4.2, PHP 7.4.30, Composer v2.1.3

**Comment by INblau-LucaGerhardt** [comment]
We went from 1.2.7 to 2.1.2.

The "magento/framework" Version in `composer.lock`, which contains the `schema.xsd` against which `db_schema.xml` is validated, is "103.0.3-p1". No custom columns for ratepay tables via `db_schema.xml` or `Install/UpgradeSchema.php` either.

The fix mentioned by @MarvinmStage is working, we applied a patch with said fix.

I'm working on WSL2 Ubuntu-20.04, Kernel version: 5.10.102.1, although I doubt that my dev setup is the cause of this. Composer Version is "2.3.0", PHP 7.4.
